src/main/java/io/specto/hoverfly/junit/core/Hoverfly.java

```diff
@@ -111,12 +108,12 @@ public void start() {
             setDestination(hoverflyConfig.getDestination());
         }
 
-        if (hoverflyMode == HoverflyMode.CAPTURE) {
-            setMode(hoverflyMode);
-        }
+        setMode(hoverflyMode);
 
-        if (useDefaultSslCert) {
-            sslConfigurer.setTrustStore();
+        if (hoverflyConfig.getAuthenticationConfig().isPresent() && hoverflyConfig.getAuthenticationConfig().get().getSslCert() != null) {
```

Comment:
Make the SSL cert optional? Then you can do functional map / filter / ifPresent / orElse